src/doc/en/developer/coding_in_cython.rst

```diff
@@ -60,14 +60,19 @@ There are several ways to create and build Cython code in Sage.
           %cython
           from __main__ import testfunction
 
-#. Create an ``.spyx`` file and attach or load it from the command
-   line. This is similar to creating a ``%cython`` cell in the
-   notebook but works completely from the command line (and not from
-   the notebook).
+#. Create an ``.spyx`` file and :ref:`attach or load it <section-attach-or-load-spyx-files>`
+   from the :ref:`command line <section-command-line>`.
+   This is similar to creating a ``%cython``
+   cell in the notebook but works completely from the command line
+   (and not from the notebook).
+
+#. Use ``%%cython`` cell magic in the command line.
+   Refer to :meth:`sage.repl.ipython_extension.SageMagics.cython`.
 
 #. Create a ``.pyx`` file and add it to the Sage library.
    Then run ``sage -b`` to rebuild Sage.
 
+.. _section-attach-or-load-spyx-files:
 
 Attaching or loading .spyx files
 ================================
```

Comment:
While we are here, we should consider moving everything about ".spyx" here into the head of https://doc-pr-38943--sagemath.netlify.app/html/en/reference/repl/sage/repl/load. 

Only things about ".pyx" are concerns of sage developers.

Reply:
On the other hand I think it's valid to want to disable the preparser too. For example `%%cython` cell does not preparse (if I recalled correctly).

But okay, point noted. (as an user stumble upon that page while looking for information how to use Cython in Sage, I hardly notice it is meant for developers.)

Reply:
Actually are you sure? Then the developer guide page would have almost nothing left except maybe

> Create a .pyx file and add it to the Sage library. Then run sage -b to rebuild Sage.


and

> **Deprecation**  
> When making a backward-incompatible modification in Sage, the old code should keep working and display a message indicating how it should be updated/written in the future. We call this a deprecation.

I'm not sure how best to organize it. Maybe leaving it as is is fine — users who want to code in Cython would stumble across the developer's guide page anyway.

Reply:
On the other hand, the two links you added in lines starting line 63 seem to be pointless. I suggest to leave the paragraph
```
#. Create an ``.spyx`` file and attach or load it from the command
   line. This is similar to creating a ``%cython`` cell in the
   notebook but works completely from the command line (and not from
   the notebook).
```
as it is, but just update `%cython` with  `%%cython`. I think `%cython` worked for old sage notebook.

Reply:
`attach()` doesn't work on the notebook though.

My issue with the command-line is the bash command-line is also called command-line (https://doc.sagemath.org/html/en/reference/repl/options.html) so…

Reply:
> `attach()` doesn't work on the notebook though. (apparently known issue from long ago, https://ask.sagemath.org/question/36364/attachattach-does-not-work-in-sage-jupyter-sage-751/ )

The notebook refers to the old (now deprecated) classic sage notebook, not the jupyter notebook. The paragraph itself is somewhat outdated. But, as I said above, updating the paragraph is not in the scope of your pr.

> My issue with the command-line is the bash command-line is also called command-line (https://doc.sagemath.org/html/en/reference/repl/options.html) so…

Then explicitly stating "the Sage command line" or "the Sage prompt" would make it less confusing. If you state "the Sage command line"  early in the paragraphs, there would really be no confusion.